.github/workflows/ts-packages.yml

```diff
@@ -60,6 +60,53 @@ jobs:
           ENABLE_GAS_REPORT: 1
       - name: Print gas report
         run: cat packages/contracts/gas-report.txt
+
+  test-coverage:
+    name: Generate test coverage
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Fetch history
+        run: git fetch
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
```

Comment:
We should consider upgrading to `14.x` or if its possible to specify LTS since `12.x` is no longer the active release, see https://nodejs.org/en/about/releases/

Reply:
LTS support has been an [open issue](https://github.com/actions/setup-node/issues/26) for quite a while. So I think 14.x is likely the best we can do.